mautic/mautic
# decode special characters in report display and export and form export and contact export

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | X
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Required: )
#### Description:
In report , contact and form export the special characters are not decoding because when a value is send in database, this value is encode and when this value is use in a page, it is decode but not in report and form export.

[//]: # ( As applicable: )
#### Steps to reproduce the bug in report:
1. Create a lead with a special character ( ex: &)
2. Use this lead in report
3. See the character encode in the report

#### Steps to test this PR:
1. Apply this PR
2. Create a lead with a special character ( ex: &)
3. Use this lead in report
4. See the character decode in the report

[//]: # ( As applicable: )
#### Steps to reproduce the bug in form export:
1. Change lead in form report with a special character ( ex: &)
2. Export the result of the form in CSV or Excel
3. See the character encode in the export

#### Steps to test this PR:
1. Apply this PR
2. Export again the result of the form in CSV or Excel
3. See the character decode in the export

[//]: # ( As applicable: )
#### Steps to reproduce the bug in contact export:
1. Change lead in form report with a special character ( ex: &)
2. Export this contact
3. See the character encode in the export

#### Steps to test this PR:
1. Apply this PR
2. Export this contact (create before with a special character)
3. See the character decode in the export
